app/src/main/java/com/github/polypoly/app/menu/ProfileActivity.kt

```diff
@@ -93,15 +101,23 @@ class ProfileActivity : ComponentActivity() {
             }
             Spacer(modifier = Modifier.height(30.dp))
             Button(
+                elevation = ButtonDefaults.elevation(
+                    defaultElevation = 0.dp,
+                    pressedElevation = 0.dp,
+                    disabledElevation = 0.dp
+                ),
                 onClick = {
                     val profileModifyingIntent = Intent(mContext, ProfileModifyingActivity::class.java)
                     profileModifyingIntent.putExtra("userId", userId)
                     startActivity(profileModifyingIntent)
                 },
                 shape = CircleShape,
-                modifier = Modifier.testTag("modifyProfileButton"),
+                modifier = Modifier
+                    .testTag("modifyProfileButton")
```

Comment:
Also, use the `testTag` only when it is really the only option to test a composable. Here it is not the case as you could use `composeTestRule.onNodeWithText("Modify profile")`.

Reply:
Okay, I agree that these strings we've been using for UI so far is a bad habit. I'll create an issue to create an external file with all the games strings.